at_commons/lib/src/keystore/at_key.dart

```diff
@@ -9,6 +9,8 @@ class AtKey {
   String? namespace;
   Metadata? metadata;
   bool isRef = false;
+  /// When set to true, represents the local key
```

Comment:
Need to explain what a local key is

Reply:
e.g. 'Local' keys are keys which will never be synced to the server. By definition they may only be 'self' keys.

Reply:
Updated the documentation